### PR TITLE
RangeWidget integer no appearance

### PR DESCRIPTION
**Pull request by grzesiek2010**
Form for testing: 
[range.txt](https://github.com/opendatakit/collect/files/1057983/range.txt)

### CODE REVIEW

**Comment by lognaturel** [comment]
This looks fantastic and works really well!

Couple of things:
- The numbers at the beginning and end of the range look good but I think they're going to be very small for users to read. I think they should be bigger. @shobhitagarwal1612, as we're reworking the UI, maybe one thing we should document is a minimum font size? I also wonder whether the tick marks might be hard to see.

- Going left to get smaller numbers tends to swipe left. It would be nice if that could be suppressed.

- There's a gray shadow that appears when dragging the slider. Sometimes it disappears when I let go of the slider, sometimes it doesn't. I would expect it to disappear.

**Comment by grzesiek2010** [comment]
@lognaturel 

> The numbers at the beginning and end of the range look good but I think they're going to be very small for users to read. I think they should be bigger. @shobhitagarwal1612, as we're reworking the UI, maybe one thing we should document is a minimum font size? I also wonder whether the tick marks might be hard to see.

ok so I'll wait for your decision about a minimum font size I think it's a good idea.

> Going left to get smaller numbers tends to swipe left. It would be nice if that could be suppressed.

I'm not sure if I understand. Are you talking about the situation when a bigger number is on the left side?
`Range7 from attached test form` is it wrong approach?

> There's a gray shadow that appears when dragging the slider. Sometimes it disappears when I let go of the slider, sometimes it doesn't. I would expect it to disappear.

something strange and pretty hard to reproduce. I'll try to find out what's wrong.

**Comment by lognaturel** [comment]
Regarding swiping left: let's say you have a slider with values 0-10 and the currently-selected value is 10. Tap and hold the slider while you slide your finger left until you go down to select the value 2. When you let go, you will also go to the next question.

File: collect_app/src/main/java/org/odk/collect/android/widgets/RangeWidget.java [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            }
+
+            currentValue = (TextView) view.findViewById(R.id.currentValue);
+            currentValue.setText(String.valueOf(actualValue));
+
+            setupSeekBar();
+        }
+    }
+
+    private void setupDefaultValues() {
```
Comment (dcbriccetti) [inline collect_app/src/main/java/org/odk/collect/android/widgets/RangeWidget.java]: “set up”, as used here, is a phrasal verb, and therefore two words. Turned into an identifier, it should be setUp, not setup.
https://www.englishclub.com/vocabulary/phrasal-verbs-list.htm

**Comment by grzesiek2010** [comment]
@lognaturel please review again

**Comment by lognaturel** [comment]
Very nice! I'll file an issue to support doubles. Of course, it would be nice to avoid code duplication between the two as much as possible.

**Comment by grzesiek2010** [comment]
> I'll file an issue to support doubles

@lognaturel it's not necessary I know what's next: appearances for existed integer widget like mentioned here https://github.com/opendatakit/xforms-spec/issues/111#issuecomment-299277448 and doubles as you said plus appearances for those as well.

**Comment by lognaturel** [comment]
Isn't it nice to close issues?! Plus, it makes it easier for other people to follow along.

**Comment by grzesiek2010** [comment]
@lognaturel 
if you prefer a few separate issues it's ok.

**Pull request merged**